apps/explorer/src/hooks/useFormatCoin.ts

```diff
@@ -105,19 +105,19 @@ export function useFormatCoin(
     );
 
     const [decimals, queryResult] = useCoinDecimals(coinType);
-    const { isFetched, isError } = queryResult;
+    const { isFetched } = queryResult;
 
     const formatted = useMemo(() => {
         if (typeof balance === 'undefined' || balance === null) return '';
 
-        if (isError) {
+        if (!coinType) {
```

Comment:
I imagine we probably want to handle both the error case and the missing coin Type case.

Reply:
In the error case, an error message will be printed to the terminal. Is there anything additional that you would like?

Reply:
Actually it looks like the decimals fetching is only enabled based on coin type, so you really shouldn't need this special casing. You probably just need to remove the error case?

Reply:
I agree with removing the error case.